dev-util/rpi-imager/rpi-imager-1.8.5.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Raspberry Pi Imaging Utility"
+HOMEPAGE="
+https://www.raspberrypi.com/software/
+https://www.raspberrypi.com/documentation/computers/getting-started.html#raspberry-pi-imager
+https://github.com/raspberrypi/rpi-imager
+"
+SRC_URI="https://github.com/raspberrypi/rpi-imager/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P}/src"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+
+KEYWORDS="~amd64"
```

Comment:
Add USE flags to handle conditional dependencies. Without these flags and src_configure, this ebuild is non-deterministic.
```
IUSE="gnutls qt6"
```